collections/ansible_collections/cloudkit/config_as_code/README.md

```diff
@@ -257,6 +269,14 @@ spec:
                 secretKeyRef:
                   name: fulfillment-admin-password
                   key: password
+            - name: CONFIG_AS_CODE_EXTRA_VARS_PATH
+              value: /var/secrets/config-as-code/vars.yaml
```

Comment:
This section should be updated to document the format of the secret. This value implies that the secret has a single key, `vars.yaml`, that contains the content from "option 1", but you're never explicit on that point. Showing how to create the secret from the file created earlier might be a good idea (e.g., showing the `oc create secret generic config-as-code --from-file=...` command line).